paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -117,6 +137,120 @@ internal class PaymentSheetPlaygroundViewModel(
         }
     }
 
+    @OptIn(ExperimentalCustomerSheetApi::class)
+    private fun prepareCustomer(
```

Comment:
This isn't making sense to me. I'd expect us to call the checkout endpoint before calling fetchEphemeralKey. What am I missing?

Or maybe this code is just significantly different when what we do in the existing customer sheet playground.

Reply:
It's a big implementation difference between our checkout and wallet flows. `PaymentSheet`/`FlowController` require that you pass in the `PaymentIntent`/`SetupIntent` ID that you already created. `CustomerSheet`, mainly the `StripeCustomerAdapter`, fetches the customer ephemeral key and creates the `SetupIntent` internally, the user just has to provide a `CustomerEphemeralKeyProvider` & `SetupIntentClientSecretProvider`.

Reply:
Chatted offline, this is using new endpoints from the existing customer sheet playground, and in turn has different logic.